### PR TITLE
Feature/company dashboard

### PR DESCRIPTION
**Pull request by AtmegaBuzz**
## Completed
- Company dashboard sidebar and navbar init design
- Analytics page UI + Backend logic
- Domain: 
    - delete domain: done
    - add_domain: done
    - list_domain: done
    - validation logic for domain creation only by domain email
    - only domain emails can be added as domain managers logic

### CODE REVIEW

**Comment by AtmegaBuzz** [comment]
@DonnieBLT you can merge it to prod for testing

File: blt/settings.py [hunk 1]
```diff
@@ -300,11 +301,15 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
-STATIC_ROOT = os.path.join(PROJECT_ROOT, "staticfiles")
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
```
Comment (DonnieBLT) [inline blt/settings.py:304]: why did this change?

File: website/alternative_views.py [hunk 2]
```diff
@@ -314,3 +321,102 @@ def get_context_data(self, **kwargs):
...
+class CreateHunt2(TemplateView):
+    model = Hunt
+    fields = ["url", "logo", "domain", "plan", "prize", "txn_id"]
+    template_name = "hunt2.html"
+
+    @method_decorator(login_required)
+    def get(self, request, *args, **kwargs):
+        
+        return render(request, self.template_name)
+        # except Exception as e:
```
Comment (DonnieBLT) [inline website/alternative_views.py:335]: if this code is no longer used, you can delete it

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
two small issues

File: website/alternative_views.py (hunk 2 above)
Comment (AtmegaBuzz) [reply website/alternative_views.py:335]: This code is for creating bughunts hunts new page which i will work on after this pr

File: blt/settings.py (hunk 1 above)
Comment (AtmegaBuzz) [reply blt/settings.py:304]: Done

**Review by DonnieBLT — APPROVED** [review]

**Comment by DonnieBLT** [comment]
nice work!

**Pull request merged**